config/client_config/dc/system/config/templates/features/enroll_app/enroll_app.yml

```diff
@@ -415,6 +415,8 @@ registry:
     features:
       - key: :generic_redirect
         is_enabled: <%= ENV['GENERIC_REDIRECT_PAY_NOW_IS_ENABLED'] || false %>
+      - key: :strict_generic_redirect
```

Comment:
Since the generic_redirect and strict_generic_redirect features are inherently coupled, we've decided to move the strict_generic_redirect key to a setting nested in the generic_redirect feature.